learn/app-development/widgets/datalive/form/form-field-validator.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Validator Function"
+id: ""
+sidebar_label: "Field Validations"
+---
+---
+
+Apply validations to a single field and multiple fields using the specified methods. These methods on the form field can be used to set sync validations on a field. These validations can include default validators like required, min value, max value, or add more customizations to the basic validations.
```

Comment:
The setup has to be little better. Start by recounting when this feature should be used